internal/metarepos/reportcommit_test.go

```diff
@@ -272,7 +272,13 @@ func TestReportCommit(t *testing.T) {
 
 				knownVersions.Lock()
 				for lsid, ver := range knownVersions.vers {
-					rsp.UncommitReports = append(rsp.UncommitReports, reportsCommits[lsid][ver].report)
+					var report snpb.LogStreamUncommitReport
+					if rcs := reportsCommits[lsid]; len(rcs) <= int(ver) {
+						report = rcs[len(rcs)-1].report
```

Comment:
In this case, wouldn't it be better to skip it? Or check if it's included in the UncommitResults that was already added?